_data/mentors.yml

```diff
@@ -181,6 +181,10 @@
   github: ananyo2012
   image: MaitiAnanyo.jpg
   twitter: MaitiAnanyo
+  
+- name: Pawan Pal
+  github: Pawan Pal
```

Comment:
Actually I see what happened. The username is highlighted -- it's also in the URL! 

![image](https://cloud.githubusercontent.com/assets/23088/21741279/87d50b5a-d49e-11e6-8149-7a9057d54c02.png)